lib/managers/webhooks.js

```diff
@@ -289,6 +289,14 @@ Webhooks.prototype.delete = function(webhookID, callback) {
  * @returns {boolean} - True or false
  */
 Webhooks.validateMessage = function(body, headers, primarySignatureKey, secondarySignatureKey, maxMessageAge) {
+	if (!primarySignatureKey && Webhooks.primaryKey) {
```

Comment:
We should add these properties to the Webhooks object at the top of the file (but set them to `null` and add JSDoc for them so it's more obvious what they're doing.